dali/python/nvidia/dali/_multiproc/shared_queue.py

```diff
@@ -159,7 +162,7 @@ def get(self, num_samples=1, predicate=None) -> Optional[List[MSG_CLASS]]:
             The call returns None iff the queue was closed.
         predicate : a parameterless callable
             Used for double-checking if the item should really be taken after waiting on empty queue.
-        """
+        """ # noqa W501
```

Comment:
My run of `flake8` didn't show `W501` violation. I suppose you meant `E501`, but in this case I'd like to ask, that you apply the suitable line breaks instead of turning off the check.